shell/browser/browser_mac.mm

```diff
@@ -532,4 +532,34 @@ void RemoveFromLoginItems() {
   }
 }
 
+void Browser::RegisterForRemoteNotifications() {
+  [[AtomApplication sharedApplication]
+      registerForRemoteNotificationTypes:NSRemoteNotificationTypeBadge |
```

Comment:
Attempting to use the updated API ([registerForRemoteNotifications](https://developer.apple.com/documentation/uikit/uiapplication/1623078-registerforremotenotifications?language=objc)) resulted in this error:

```
'registerForRemoteNotifications' has been marked as being introduced in macOS 10.14 here, but the deployment target is macOS 10.11.0
```